lib/pure/asyncdispatch.nim

```diff
@@ -101,16 +101,9 @@
 ## Handling Exceptions
 ## -------------------
 ##
-## The most reliable way to handle exceptions is to use `yield` on a future
-## then check the future's `failed` property. For example:
-##
-## .. code-block:: Nim
-##   var future = sock.recv(100)
-##   yield future
-##   if future.failed:
-##     # Handle exception
+## You can handle exceptions in the same way as in ordinary Nim code;
+## by using the try statement:
```

Comment:
It would be worth noting that the effect system does not work across await boundaries